src/pages/typescript/index.md

```diff
@@ -1,12 +1,37 @@
 ---
-title: Typescript
+title: TypeScript
 ---
-## Typescript
+## TypeScript
 
-TypeScript is a typed superset of JavaScript that compiles to plain JavaScript.
+### Overview
 
-Also, it adds static typing functionality to JavaScript.
+![TypeScript](https://i.imgur.com/uRLunzU.png)
 
-If you want to try out TypeScript without installation, visit the <a href='http://www.typescriptlang.org/play/index.html' target='_blank' rel='nofollow'>TypeScript Playground</a>. 
+So as you are most likely aware, JavaScript is expanding its footprint everyday and it is both overwhelming and amazing what you can do with the language nowadays.
 
-To learn more about installation, see the [Installation Appendix](./src/articles/typescript/appendix-installation/index.md).
+However, as more large-scale projects start to use JavaScript, the process of making the code easier to write and more maintainable becomes more and more difficult.
+
+This is a problem Microsoft recognized early on and they came up with the solution of TypeScript and released the first version approximately on October 1st, 2012.
+
+![Key Features](https://i.imgur.com/pZij95O.jpg)
+
+In essence, TypeScript is trying to help JavaScript reach new heights and become very scalable and can be highlighting by the following features:
+
+- free and open-source programming language developed and maintained by Microsoft
+- strict syntactical super-set of JavaScript that compiles to plain JavaScript
+- eases development of large scale applications written in JavaScript
+- extends JavaScript by adding static types, classes, modules, interfaces and generics
+
+**🎉 FUN FACT** TypeScript turned 5 years old on October 1st, 2017.
+
+### Installation
+
+To learn more about installation, see [Installation](https://guide.freecodecamp.org/typescript/installation).
+
+### Playground
+
+![Playground](https://i.imgur.com/vlV7ZFr.png)
+
+If you want to try out TypeScript without installing it, visit the <a href='http://www.typescriptlang.org/play/index.html' target='_blank' rel='nofollow'>TypeScript Playground</a>.
+
+The Playground has built-in auto completion and the ability to directly see the emitted JavaScript.
```

Comment:
You could maybe add some "Other Resources" link at the end

```
### Other Resources

* <a href='http://www.typescriptlang.org/samples/index.html' target='_blank' rel='nofollow'>Quick Start</a>
* <a href='http://www.typescriptlang.org/docs/home.html' target='_blank' rel='nofollow'>Documentation</a>
* <a href='https://github.com/Microsoft/TypeScript' target='_blank' rel='nofollow'>Source Code</a>
```

Reply:
Added!